test/runtests.jl

```diff
@@ -147,6 +148,13 @@ AstronomicalTime.update()
         @test ref == TDBEpoch(TCBEpoch("2013-03-18T12:00:17.718"))
         @test ref == TDBEpoch(TCGEpoch("2013-03-18T12:00:00.795"))
     end
+    # Test of Ported function
+    @testset "PortedFunctions" begin
+        tt = TTEpoch(2000, 1, 1, 12, 0, 0.0)
+        @test taitt(julian1(tt), julian2(tt)) == eraTaitt(julian1(tt), julian2(tt))
+        @test taitt(julian2(tt), julian1(tt)) == eraTaitt(julian2(tt), julian1(tt))
+        @test TTEpoch(taitt(julian2(tt), julian1(tt))...) ≈ TTEpoch(taitt(julian1(tt), julian2(tt))...)
```

Comment:
This test does not give us any new information, please remove it. I was surprised that it passes, though. We might want to change this behaviour in the future.